llvm/tools/sycl-post-link/CompileTimeProperties.def

```diff
@@ -16,3 +16,16 @@
 // https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/DeviceGlobal/SPV_INTEL_global_variable_decorations.asciidoc#decoration
 SYCL_COMPILE_TIME_PROPERTY("sycl-init-mode", 6148, DecorValueTy::uint32)
 SYCL_COMPILE_TIME_PROPERTY("sycl-implement-in-csr", 6149, DecorValueTy::boolean)
+
```

Comment:
Hmm. I am a bit confused about all these entries. I was expecteing only one entry for the 'sycl-alignment'.

Thanks

Reply:
Done. This is resolved since another patch has been merged so this is not needed anymore